src/components/NormalRightRow/index.jsx

```diff
@@ -3,17 +3,21 @@ import { shape, string } from 'prop-types';
 
 function NormalRightRow({ schema, classes }) {
   const keywords = Object.keys(schema);
+  const nonDisplayedKeywords = ['items'];
 
   return (
     <div className={`${classes.row} ${classes.rightRow}`}>
       <div className={classes.keywordColumn}>
-        {keywords.map(keyword => (
-          <p key={keyword} className={classes.line}>
-            {keyword}
-            {': '}
-            {schema[keyword]}
-          </p>
-        ))}
+        {keywords.map(
+          keyword =>
+            !nonDisplayedKeywords.includes(keyword) && (
+              <p key={keyword} className={classes.line}>
```

Comment:
Rather than using `p`, we can use `Typography`. Also, should this be a paragraph or more like a span/div?

```jsx
<Typography component="span">
  ...
</Typography>
```

Reply:
Great idea on using `Typography`! 👍 

Since each keyword should maintain its own line (should be a block component like `p` or `div`, rather than a inline component like `span`), I'll change the `p` to using the following:
```html
<Typography component="div">
  ...
</Typography>
```

Reply:
Applied changes!